certbot/tests/ocsp_test.py

```diff
@@ -1,18 +1,34 @@
 """Tests for ocsp.py"""
 # pylint: disable=protected-access
-
 import unittest
+from datetime import datetime, timedelta
 
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes  # type: ignore
+from cryptography.exceptions import UnsupportedAlgorithm, InvalidSignature
+from cryptography import x509
+try:
+    # Only cryptography>=2.5 has ocsp module
+    # and signature_hash_algorithm attribute in OCSPResponse class
+    from cryptography.x509 import ocsp as ocsp_lib  # pylint: disable=import-error
+    from cryptography.x509.ocsp import OCSPResponse  # pylint: disable=import-error,unused-import
```

Comment:
nit: Can we remove this line and just use `ocsp_lib.OCSPResponse` in the line below?